include/SSVOpenHexagon/Components/CPlayer.hpp

```diff
@@ -5,6 +5,7 @@
 #pragma once
 
 #include "SSVOpenHexagon/Utils/Ticker.hpp"
+#include "CWall.hpp"
```

Comment:
We don't need a physical dependency on `CWall.hpp` here, as we are only using `CWall` by reference (which does not require its definition). A [forward declaration](https://stackoverflow.com/questions/4757565/what-are-forward-declarations-in-c) would suffice. See line 18 in this file for an example. Reducing physical dependencies avoids dependency cycles and speeds up compilation time.